src/Translator.php

```diff
@@ -88,7 +92,7 @@ private function checkEnv(): void
         if (strpos($this->txSite, 'http://') !== 0 && strpos($this->txSite, 'https://') !== 0) {
             $this->txSite = 'http://' . $this->txSite;
         }
-        if (!filter_var($this->txSite, FILTER_VALIDATE_URL)) {
+        if (!parse_url($this->txSite, PHP_URL_HOST)) {
```

Comment:
`filter_var` with `FILTER_VALIDATE_URL` uses [RFC2396](http://www.faqs.org/rfcs/rfc2396.html) which is obsolete.
`parse_url()` uses [RFC3986](http://www.faqs.org/rfcs/rfc3986.html) which is what is currently in effect.
Or at least that's what I read on stackoverflow - in either case `parse_url()` correctly identifies `http://textcollection_08.local` as being a valid URL.